### PR TITLE
Add rspec documentation and apitome

### PR DESCRIPTION
**Pull request by LeandroSoria2**
### What changes does this PR include?
**Description**
Add gem apitome and rspec documentation, and set up 
**Summary**
-Add apitome and config
-Add rspec documentation 

**Trello**
(https://trello.com/c/TB9ZRVMT/32-rspec-documentation)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,12 +46,14 @@ group :development, :test do
...
   gem 'factory_bot_rails', '~> 6.2'
   gem 'faker', '~> 2.21'
+  gem 'rspec_api_documentation'
   gem 'rspec-rails', '~> 5.1', '>= 5.1.2'
 end
 
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'annotate', '~> 3.2'
+  gem 'apitome', '~> 0.3.1'
```
Comment (AgostinaDev) [inline Gemfile:56]: Looks like tests are failing because this gem is not loading in the test environment

Comment (AgostinaDev) [reply Gemfile:56]: You can prevent Apitome from loading [like this](https://github.com/rootstrap/forecast/blob/8f97bcfe9795cdde6a030b535a6998e86f86183a/config/initializers/apitome.rb#L1)

Comment (LeandroSoria2) [reply Gemfile:56]: I will change the group

File: Gemfile [hunk 2]
```diff
@@ -43,9 +43,11 @@ gem 'sass-rails', '~> 6.0'
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'apitome', '~> 0.3.1'
```
Comment (AgostinaDev) [inline Gemfile]: It's ok to load this gem in development environments only, you just have to adjust the code [like I mentioned before in this comment](https://github.com/LeandroSoria2/TargetApi/pull/19#discussion_r920245084)

**Review by AgostinaDev — APPROVED** [review]
Looks good to me :)

**Review by JulianPasquale — APPROVED** [review]

**Pull request merged**